### PR TITLE
Homework16.2

### PR DESCRIPTION
**Pull request by JenyaKh**

### CODE REVIEW

File: lms/settings.py [hunk 1]
```diff
@@ -89,10 +89,18 @@
     }
 }
 
+EMAIL_BACKEND = 'django.core.mail.backends.smtp.EmailBackend'
```
Comment (mikolaz27) [inline lms/settings.py:92]: I got an error when try to reset password.
image

File: students/views.py [hunk 2]
```diff
@@ -1,105 +1,161 @@
...
+    email.send()
+    return HttpResponse('Done')
+
+
+def password_reset_request(request):
+    if request.method == "POST":
+        password_reset_form = PasswordResetForm(request.POST)
+        if password_reset_form.is_valid():
+            data = password_reset_form.cleaned_data['email']
+            associated_users = User.objects.filter(Q(email=data))
```
Comment (mikolaz27) [inline students/views.py]: email should be uniq, no reason to use this for. And also Q is not needed here.

File: students/views.py [hunk 3]
```diff
@@ -1,105 +1,161 @@
...
+            if associated_users.exists():
+                for user in associated_users:
+                    subject = "Password Reset Requested"
+                    email_template_name = "registration/password_reset_email.txt"
```
Comment (mikolaz27) [inline students/views.py]: why txt? it should be html

File: lms/settings.py (hunk 1 above)
Comment (JenyaKh) [reply lms/settings.py:92]: I can't reproduce this error (( Added @csrf_exempt to the view function

**Pull request merged**